lua52-sys/src/lib.rs

```diff
@@ -27,6 +27,7 @@ pub const LUA_ERRERR: c_int = 6;
 pub struct lua_State;
 
 pub type lua_CFunction = extern "C" fn(L: *mut lua_State) -> c_int;
+pub type lua_CFunction2 = unsafe extern "C" fn(*mut lua_State);
```

Comment:
I'm not sure why I needed to create this type, I see other FFI declerations use the `lua_CFunction` type, and the docs for [`requiref`](https://www.lua.org/manual/5.2/manual.html#luaL_requiref) don't seem any different to me.